bot/cogs/help.py

```diff
@@ -70,499 +68,307 @@ class HelpSession:
         else:
             return "**\u200bNo Category:**"
 
-    def _get_command_params(self, cmd: Command) -> str:
+    async def send_category_help(self, category: Category) -> None:
         """
-        Returns the command usage signature.
+        Sends help for a bot category.
 
-        This is a custom implementation of `command.signature` in order to format the command
-        signature without aliases.
+        This sends a brief help for all commands in all cogs registered to the category.
         """
-        results = []
-        for name, param in cmd.clean_params.items():
-
-            # if argument has a default value
-            if param.default is not param.empty:
-
-                if isinstance(param.default, str):
-                    show_default = param.default
-                else:
-                    show_default = param.default is not None
+        embed = Embed()
+        embed.set_author(name="Command Help", icon_url=constants.Icons.questionmark)
 
-                # if default is not an empty string or None
-                if show_default:
-                    results.append(f'[{name}={param.default}]')
-                else:
-                    results.append(f'[{name}]')
-
-            # if variable length argument
-            elif param.kind == param.VAR_POSITIONAL:
-                results.append(f'[{name}...]')
-
-            # if required
-            else:
-                results.append(f'<{name}>')
-
-        return f"{cmd.name} {' '.join(results)}"
-
-    async def build_pages(self) -> None:
-        """Builds the list of content pages to be paginated through in the help message, as a list of str."""
-        # Use LinePaginator to restrict embed line height
-        paginator = LinePaginator(prefix='', suffix='', max_lines=self._max_lines)
-
-        prefix = constants.Bot.prefix
+        all_commands = []
+        for c in category.cogs:
+            all_commands.extend(c.get_commands())
 
-        # show signature if query is a command
-        if isinstance(self.query, commands.Command):
-            signature = self._get_command_params(self.query)
-            parent = self.query.full_parent_name + ' ' if self.query.parent else ''
-            paginator.add_line(f'**```{prefix}{parent}{signature}```**')
-
-            # show command aliases
-            aliases = ', '.join(f'`{a}`' for a in self.query.aliases)
-            if aliases:
-                paginator.add_line(f'**Can also use:** {aliases}\n')
-
-            if not await self.query.can_run(self._ctx):
-                paginator.add_line('***You cannot run this command.***\n')
-
-        # show name if query is a cog
-        if isinstance(self.query, Cog):
-            paginator.add_line(f'**{self.query.name}**')
-
-        if self.description:
-            paginator.add_line(f'*{self.description}*')
+        filtered_commands = await self.filter_commands(all_commands, sort=True, key=self._category_key)
```

Comment:
Commands weren't being sorted according to their names and I believe this simple change is the fix.

```suggestion
        filtered_commands = await self.filter_commands(all_commands, sort=True)
```